### PR TITLE
Polyvox fixes

### PR DESCRIPTION
**Pull request by sethalves**
Fix rendering positions.  A script calling setVoxel will cause an update to go over the network.  Collisions are still not quite right.

### CODE REVIEW

File: libraries/entities-renderer/src/RenderablePolyVoxEntityItem.cpp [hunk 1]
```diff
@@ -267,7 +275,7 @@ void RenderablePolyVoxEntityItem::setAll(uint8_t toValue) {
 }
 
 void RenderablePolyVoxEntityItem::setVoxelInVolume(glm::vec3 position, uint8_t toValue) {
-    updateOnCount(position.x, position.y, position.z, toValue);
+    // same as setVoxel but takes a vector rather than 3 floats.
     setVoxel(position.x, position.y, position.z, toValue);
 }
```
Comment (AndrewMeadows) [inline libraries/entities-renderer/src/RenderablePolyVoxEntityItem.cpp]: setVoxel() takes int x, y, and z, so there are some implicit casts happening here that should be made explicit.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e98b28c1d8e620626afde89593aee61338dbb277 available at: https://gist.github.com/8debac1f3e54d8d810cf<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5132.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5132.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d7eddc398b5b87b73e14dab2f716d71b64877087 available at: https://gist.github.com/8debac1f3e54d8d810cf<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5132.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5132.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f5cea57a48729008e85ac853b7edd62dce8b6c69 available at: https://gist.github.com/8debac1f3e54d8d810cf<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5132.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5132.dmg

File: libraries/entities/src/PolyVoxEntityItem.h [hunk 2]
```diff
@@ -82,10 +82,11 @@ class PolyVoxEntityItem : public EntityItem {
     virtual void setAll(uint8_t toValue) {}
 
     virtual void setVoxelInVolume(glm::vec3 position, uint8_t toValue) {}
-    
+
     virtual uint8_t getVoxel(int x, int y, int z) { return 0; }
     virtual void setVoxel(int x, int y, int z, uint8_t toValue) {}
 
+    static QByteArray makeEmptyVoxelData(quint16 voxelXSize = 16, quint16 voxelYSize = 16, quint16 voxelZSize = 16);
```
Comment (PhilipRosedale) [inline libraries/entities/src/PolyVoxEntityItem.h]: magic number.  I'll merge but maybe explain for next set of changes?

**Pull request merged**